src/mesh/mesh_function.C

```diff
@@ -50,9 +50,7 @@ MeshFunction::MeshFunction (const EquationSystems & eqn_systems,
   _vector              (vec),
   _dof_map             (dof_map),
   _system_vars         (vars),
-  _point_locator       (nullptr),
-  _out_of_mesh_mode    (false),
-  _out_of_mesh_value   ()
```

Comment:
Wait a sec - doesn't this leave _out_of_mesh_value uninitialized?  I could easily imagine some user code relying on just getting 0 by default without manually saying so.

Reply:
out_of_mesh_value is a DenseVector so it will be default constructed no matter what, no reason to explicitly do so?

Reply:
You're right, never mind.